shadow-dom/event-with-related-target.html

```diff
@@ -0,0 +1,264 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Shadow DOM: Firing an event with relatedTarget inside a shadow tree</title>
+<meta name="author" title="Ryosuke Niwa" href="mailto:rniwa@webkit.org">
+<meta name="assert" content="The retargeting algorithm is used to determine relative targets">
+<link rel="help" href="http://w3c.github.io/webcomponents/spec/shadow/#dfn-retargeting-algorithm">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
```

Comment:
Apparently this needs `<script src="resources/evnet-path-test-helpers.js"></script>`.